### DISCUSSION TITLE
Thanks for the more complete registers list.

### DISCUSSION THREAD

**Discussion by leodesigner** [odya/esphome-powmr-hybrid-inverter#16] [General]
Hello from Lviv,
Linked your registers list from my readme.
Thanks for making it more complete!

**Comment by odya** [comment]
🤝🍻🇺🇦🔪🐷🐶🤙🍉😉

**Comment by leodesigner** [comment]
Would you please correct registers for PV Voltage and PV Power?
https://github.com/leodesigner/powmr_comm/blob/7b6013a9f52fbb2e9631fe641bb2b241751c08cb/src/main.cpp#L977
4504 : PV Voltage
4505 : PV Power

**Comment by odya** [comment]
Are you sure that 4505 is PV Power? I don't have a PV panel, but as far as I remember, I was receiving the onboard charger state (1/0) on this register during my tests (on HVM2.4H-24V).

**Comment by leodesigner** [comment]
Yes, I use it for my PV Panel readings.
